pinot-common/src/main/java/org/apache/pinot/common/request/context/ExpressionContext.java

```diff
@@ -31,37 +34,53 @@
  */
 public class ExpressionContext {
   public enum Type {
-    LITERAL, IDENTIFIER, FUNCTION
+    LITERAL, IDENTIFIER, FUNCTION,
   }
 
   private final Type _type;
   private final String _value;
   private final FunctionContext _function;
+  // Only set when the _type is LITERAL
+  @Nullable
```

Comment:
(minor) We don't usually put `@Nullable` in variable declaration.

Reply:
Removed.